cmd/coordinated/main.go

```diff
@@ -56,27 +56,16 @@ func main() {
 	}
 	coordinate = cache.New(coordinate)
 
-	var reqLogger *logrus.Logger
-	if *logRequests {
-		stdlog := logrus.StandardLogger()
-		reqLogger = &logrus.Logger{
-			Out:       stdlog.Out,
-			Formatter: stdlog.Formatter,
-			Hooks:     stdlog.Hooks,
-			Level:     logrus.DebugLevel,
-		}
+	reqLogger := logrus.StandardLogger()
+	if !*logRequests {
+		reqLogger.Out = ioutil.Discard
```

Comment:
I think this actually changes the behavior? In the previous code, a logger with `DebugLevel` is built, but in this code, I think the default level is used, which seems to be `InfoLevel`: https://github.com/sirupsen/logrus/blob/master/logger.go#L74 --- Is that intentional?